examples/plot_meg_data_example.py

```diff
@@ -93,34 +92,89 @@ def _compute_stc(zscore_active_set, active_set, evoked, forward):
     return stc
 
 
+# Choose the experiment (task)
+list_cond = ['audio', 'visual', 'somato']
+cond = list_cond[0]
+
 # Downloading data
-subject = 'sample'
-data_path = sample.data_path()
-fwd_fname = data_path + '/MEG/sample/sample_audvis-meg-eeg-oct-6-fwd.fif'
-ave_fname = data_path + '/MEG/sample/sample_audvis-ave.fif'
-raw_fname = data_path + '/MEG/sample/sample_audvis_raw.fif'
-cov_fname = data_path + '/MEG/sample/sample_audvis-shrunk-cov.fif'
-subjects_dir = data_path + '/subjects'
-condition = 'Left Auditory'
-
-# Read noise covariance matrix
-noise_cov = mne.read_cov(cov_fname)
-
-# Handling average file
-evoked = mne.read_evokeds(ave_fname, condition=condition, baseline=(None, 0))
-evoked = evoked.pick_types('grad')
-# Selecting relevant time window
-evoked.plot()
-evoked.crop(tmin=0.05, tmax=0.1)
-
-# Handling forward solution:
+if cond in ['audio', 'visual']:
+
+    subject = 'sample'
+    data_path = sample.data_path()
+    fwd_fname_suffix = 'MEG/sample/sample_audvis-meg-eeg-oct-6-fwd.fif'
+    fwd_fname = os.path.join(data_path, fwd_fname_suffix)
+    ave_fname = os.path.join(data_path, 'MEG/sample/sample_audvis-ave.fif')
+    raw_fname = os.path.join(data_path, 'MEG/sample/sample_audvis_raw.fif')
+    cov_fname_suffix = 'MEG/sample/sample_audvis-shrunk-cov.fif'
+    cov_fname = os.path.join(data_path, cov_fname_suffix)
+    cov_fname = data_path + '/' + cov_fname_suffix
+    subjects_dir = os.path.join(data_path, 'subjects')
+
+    if cond == 'audio':
+        condition = 'Left Auditory'
+    elif cond == 'visual':
+        condition = 'Left visual'
+
+    # Read noise covariance matrix
+    noise_cov = mne.read_cov(cov_fname)
+
+    # Handling average file
+    evoked = mne.read_evokeds(ave_fname, condition=condition,
+                              baseline=(None, 0))
+    evoked = evoked.pick_types('grad')
+
+    # Selecting relevant time window
+    evoked.plot()
+    t_min, t_max = 0.05, 0.1
+    t_step = 0.01
```

Comment:
what is this tstep? I would add a decim parameter explaining it's for speed reason in the example.

Reply:
Not sure what would be the decim paramter ? Is it the same as:

`step = int(t_step * evoked.info['sfreq'])`